src/program.js

```diff
@@ -215,6 +215,14 @@ Example: $0 --help run.
           describe: 'Number of milliseconds to wait before giving up',
           type: 'number',
         },
+        'update-link': {
+          describe:
+            'A URL to where your XPI files will be hosted. ' +
+            'ex: https://example.com/plugins/{xpiFileName} ' +
```

Comment:
New sentences should start with a capital letter. I think it would also be helpful to spell out `Example: `